pkg/controller/operators/decorators/operator.go

```diff
@@ -62,10 +63,10 @@ func OperatorNames(labels map[string]string) (names []types.NamespacedName) {
 
 type OperatorFactory interface {
 	// NewOperator returns an Operator decorator that wraps the given external Operator representation.
-	// An error is returned if the decorator cannon be instantiated.
+	// An error is returned if the decorator cannot be instantiated.
 	NewOperator(external *operatorsv1.Operator) (*Operator, error)
 
-	// NewPackageOperator returns an Operator decorator for a package and install namespace.
+	// NewPackageOperator returns an Operator decorator for a package and installs namespace.
```

Comment:
This was not a typo. It should be `install namespace`.

Reply:
Thanks for pointing it out. I reverted it back to `install`